README.md

```diff
@@ -899,6 +899,12 @@ Example of expected Vanillatweaks file format:
       "coordinates hud"
     ],
     "items": ["armored elytra"]
+  },
+  "craftingtweaks": {
```

Comment:
Are you able to add this to the test file as well for integration testing?
`./docker-minecraft-server/tests/setuponlytests/forgeapimods_file/example.json`